cognizant-studio-academy/User-Management
# Change done to Login

#### What's this PR do?
This PR contains Admin login functionalities, which has been changed.

#### Where should the reviewer start?
All files in Client/

#### How should this be manually tested? 
Step 1 : With Valid Credentials , User clicks on Login button
Step 2: After Successful login verify the Welcome page with Welcome message
Step 3: With Invalid Credentials , User will not be able to go to Welcome Page. 
 
#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/166152348
